apps/forge/src/routes/_authenticated/_reponly/sign-in/dashboard/-components/QueuedUserCard.tsx

```diff
@@ -84,7 +88,7 @@ export const QueuedUserCard: React.FC<QueuedUserCardProps> = ({ place, onDequeue
           </div>
         </div>
       </div>
-      <TimeDisplay timeIn={place.created_at ?? iForgeEpoch} inText="Queued at:" durationText={canSignIn ? "Time Left:" : "Queuing for:"} />
+      <TimeDisplay timeIn={place.ends_at ?? place.created_at} inText="Queued at:" durationText={canSignIn ? newDurationValue > 0 ? "Overdue By:" : "Time Left:" : "Queuing for:"} />
```

Comment:
This is a syntax error, you can't have a ternary like this. You need another condition to check against for 3 outputs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator

Reply:
I don't think it's a syntax error - maybe needs some brackets? i.e.
canSignIn ? { 
newDurationValue > 0 ? "Overdue By:" : "Time left:"
} : "Queuing for:"

Reply:
Oh apologies yes that does look fine now, not sure how I missed that